src/Admin/DevToolsUserAccess.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Class DevToolsUserAccess.
+ *
+ * @since 1.6.0
+ *
+ * @package AMP
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Delayed;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+use WP_Error;
+use WP_User;
+
+/**
+ * Class DevToolsUserAccess
+ *
+ * @since 1.6.0
+ */
+final class DevToolsUserAccess implements Delayed, Service, Registerable {
+
+	/**
+	 * User meta key enabling or disabling developer tools.
+	 *
+	 * @var string
+	 */
+	const USER_FIELD_DEVELOPER_TOOLS_ENABLED = 'amp_dev_tools_enabled';
+
+	/**
+	 * Get the action to use for registering the service.
+	 *
+	 * @return string Registration action to use.
+	 */
+	public static function get_registration_action() {
+		return 'rest_api_init';
```

Comment:
If not in this PR, in a subsequent PR we'll need to expand this so that we can also register a user profile field. We should perhaps not use `Delayed` in that case.

Reply:
Done in 307aa8c32291b670f821d139487f2c9282abc79a